persistence/sql/sqlserver-combining-persistence-wth-transport.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Combining MS SQL Server Persistence and Transport
+component: SqlPersistence
+reviewed: 2024-06-10
+related:
+  - persistence/sql/dialect-mysql
+  - transports/sql
+---
+
+## Connection behavior
+
+When combining SQL Server transport and persistence using the Sql dialect, the connection behaves differently based on whether the [Outbox](/nservicebus/outbox/) is used or not. This influences where the saga data is stored.
+
+### Without Outbox
+
+SQL Transport<br/>TransactionMode | SQL Persistence<br/>with Sql dialect | Connection sharing | Saga location
+:-:|:-:|:-:|:-:
+TransactionScope |  ✅| Connection sharing via SQLT storage context | Transport Db
```

Comment:
Note, I'm not sure if &lt;sup&gt; will work on docs site

```suggestion
TransactionScope |  ✅| SQLT transaction is promoted to distributed transaction | Persistence DB <sup>1</sup>
```

Reply:
@abparticular it seems to work ok - I tried it locally first then pushed the change.